opus/application/static_media/js/browse.js

```diff
@@ -978,28 +976,33 @@ var o_browse = {
 
     updateBrowseNav: function() {
         let tab = `#${opus.prefs.view}`;
```

Comment:
I thought we were going to abstract this away so that opus.prefs.view was not used directly as a selector.